cmake/BuildTests.cmake

```diff
@@ -76,10 +76,12 @@ function(createTests testName)
     endif()
     target_link_libraries(${testName} ${CMAKE_DL_LIBS} ${SPD_LIB} ${TEST_BASE_LIB})
     target_link_libraries(${testName} ${CMAKE_THREAD_LIBS_INIT}  core-minifi yaml-cpp)
-    #${OPENSSL_LIBRARIES}
     if (NOT excludeBase)
       if (APPLE)
   		target_link_libraries (${testName} -Wl,-all_load minifi)
+	  elseif(WIN32)
+	    target_link_libraries (${testName} minifi)
+	    set_target_properties(${testName} PROPERTIES LINK_FLAGS "${LINK_FLAGS} /WHOLEARCHIVE:minifi")
```

Comment:
We should use wholearchive judiciously....not sure if this is a case where this needs to be removed.

Reply:
Those symbols are in standard-processors and minifiexe. I haven't tried removing this, though, as it's low priority and frankly it's a test...so I'm not sure I really care about it.